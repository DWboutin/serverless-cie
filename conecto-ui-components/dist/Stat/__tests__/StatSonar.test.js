"use strict";

var _interopRequireDefault = require("@babel/runtime/helpers/interopRequireDefault");

var _react = _interopRequireDefault(require("react"));

var _enzyme = require("enzyme");

var _Stat = require("../../Stat");

var _Icon = _interopRequireDefault(require("../../Icon"));

describe('StatTwoNumbers', () => {
  it('should match snapshot', () => {
    const wrapper = (0, _enzyme.shallow)(_react.default.createElement(_Stat.StatSonar, {
      icon: _react.default.createElement(_Icon.default, {
        type: "notification"
      }),
      title: "Taux d'accceptation",
      notice: "+ 6.5% / mois pr\xE9c\xE9dent",
      number: 60,
      suffix: "h"
    }));
    expect(wrapper).toMatchSnapshot();
  });
  it('should have the prefix class', () => {
    const wrapper = (0, _enzyme.shallow)(_react.default.createElement(_Stat.StatSonar, {
      icon: _react.default.createElement(_Icon.default, {
        type: "notification"
      }),
      title: "Taux d'accceptation",
      notice: "+ 6.5% / mois pr\xE9c\xE9dent",
      number: 60,
      suffix: "h"
    }));
    expect(wrapper.hasClass('to-stat-sonar')).toEqual(true);
  });
});