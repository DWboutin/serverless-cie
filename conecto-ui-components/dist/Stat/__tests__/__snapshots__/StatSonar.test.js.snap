// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StatTwoNumbers should match snapshot 1`] = `
<CardStat
  className="to-stat-sonar"
  icon={
    <Icon
      type="notification"
    />
  }
  title="Taux d'accceptation"
  type="bordered"
>
  <div
    className="sonar"
  >
    <div
      className="inner-sonar"
    >
      <div
        className="number"
      >
        <span>
          <CountUp$$1
            decimal="."
            decimals={0}
            delay={null}
            duration="2"
            easingFn={null}
            end={60}
            formattingFn={null}
            onEnd={[Function]}
            onPauseResume={[Function]}
            onReset={[Function]}
            onStart={[Function]}
            onUpdate={[Function]}
            prefix=""
            redraw={false}
            separator=""
            start={0}
            suffix=""
            useEasing={true}
          />
          h
        </span>
      </div>
    </div>
    <div
      className="icon-sonar"
    >
      <div
        className="icon-sonar-wrap"
      >
        <span>
          <Icon
            type="marker"
          />
        </span>
      </div>
    </div>
  </div>
</CardStat>
`;
